src/uwtools/tests/drivers/test_driver.py

```diff
@@ -56,23 +58,24 @@ def driverobj(tmp_path):
                     "batchargs": {
                         "export": "NONE",
                         "nodes": 1,
-                        "stdout": "/path/to/file",
+                        "stdout": "{{ concrete.run_dir }}/out",
                         "walltime": "00:05:00",
                     },
                     "executable": "qux",
                     "mpiargs": ["bar", "baz"],
                     "mpicmd": "foo",
                 },
-                "run_dir": "/path/to/dir",
+                "run_dir": "{{ rootdir }}/{{ cycle.strftime('%Y%m%d%H') }}/run",
```

Comment:
Exercise the combination of an internal reference and the driver-supplied `cycle` value.